renderer/src/components/Calendar/Calendar.tsx

```diff
@@ -303,6 +356,85 @@ export function Calendar({
             nextCallback={nextButtonHandle}
           />
         </div>
+        {screenWidth >= SCREENS.LG && (
+          <Hint
+            displayCondition={true}
+            learningMethod="nextClick"
+            order={1}
+            groupName="calendar"
+            referenceRef={allCalendarRef}
+            shiftY={150}
+            shiftX={50}
+            width={"medium"}
+            position={{
+              basePosition: "right",
+              diagonalPosition: "top",
+            }}
+          >
+            Within the calendar, you can easily track the time you've reported
+            for each day. It provides visibility into your vacation days, sick
+            leave, and holidays, along with identifying reports containing
+            errors. The current day is highlighted in yellow, while the day
+            you've selected for viewing is marked in blue.
+          </Hint>
+        )}
+        {screenWidth < SCREENS.LG && (
+          <Hint
+            displayCondition={true}
+            learningMethod="nextClick"
+            order={1}
+            groupName="calendar"
+            referenceRef={allCalendarRef}
+            shiftY={50}
+            shiftX={0}
+            width={"medium"}
+            position={{
+              basePosition: "top",
+              diagonalPosition: "right",
+            }}
+          >
+            Within the calendar, you can easily track the time you've reported
```

Comment:
Could you please move this text to the constants file and use it from that? And, please, do it for all texts for Hints.

Reply:
You could move it to general constants file in helpers folder. 
It should be named as `export const HINTS_ALERTS = {}`